runtime/domain.c

```diff
@@ -1227,13 +1227,9 @@ CAMLexport void caml_process_pending_actions(void)
   caml_process_pending_signals();
 }
 
-void caml_handle_gc_interrupt_no_async_exceptions(void)
-{
-  handle_gc_interrupt();
-}
-
 void caml_handle_gc_interrupt(void)
 {
+  /* FIXME: do not run async callbakcs */
```

Comment:
Typo in callbakcs.

Reply:
Ok, it happens to be deleted in subsequent commit